x/crosschain/types/utils.go

```diff
@@ -0,0 +1,77 @@
+package types
```

Comment:
Can we avoid using `utils` and put these methods in file reflecting the functionalities?

Reply:
These are just utility functions, mostly used during cctx validation, but can be used by other functions in the future maybe for message.Validate() . 
I don't really ,mind changing it , if you have a filename for these functions in mind please lmk

Reply:
Validation messages in `validate.go`
Authz messages in `authz.go`

Reply:
2c856b06d8026602ae5118c27cf657bf42a04bb9